docs/import-etl.rst

```diff
@@ -5,11 +5,11 @@ Import / ETL
 ###############################
 GenomicsDB supports importing genomics data in several common formats, and with a variety of methods.
 
-GenomicsDB can ingest data in VCF, gVCF. and CSV formats. When importing VCFs or gVCFs, 
+GenomicsDB can ingest data in VCF, gVCF, BCF, gBCFand CSV formats. When importing VCFs or gVCFs,
```

Comment:
```suggestion
GenomicsDB can ingest data in VCF, gVCF, BCF, gBCF and CSV formats. When importing VCFs or gVCFs,
```
missing a space. Also, is there such a thing as a gBCF?? TIL!

Reply:
No idea. Just picked it up from later in the document - see https://genomicsdb.readthedocs.io/en/latest/import-etl.html#vcf-gvcf. I can remove it if there is no such beast!